plugins/acss-kit/assets/utilities/utilities.css

```
@@ -0,0 +1,1436 @@
+.xl-gap-24 { gap: 6rem; }
+.xl-gap-32 { gap: 8rem; }
+}
+
+/* display utilities */
+.hide { display: none !important; }
+.show { display: revert !important; }
+.invisible { visibility: hidden !important; }
+.sr-only {
+  position: absolute; width: 1px; height: 1px;
+  padding: 0; margin: -1px; overflow: hidden;
+  clip: rect(0, 0, 0, 0); white-space: nowrap; border-width: 0;
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)